### DISCUSSION TITLE
Permissions issue

### DISCUSSION THREAD

**Discussion by pinkairfreshener** [jlesage/docker-filebot#97] [Q&A] [answered]
I am running filebot in docker compose. COntainer specified to run on my user (1002:1002). The volumes I have bind mounted have read and write permission by 1002. COntainer started and there is an eror. Logs show:

[cont-init   ] 10-init-users.sh: executing...
[cont-init   ] 10-init-users.sh: rm: cannot remove '/etc/passwd': Permission denied
[cont-init   ] 10-init-users.sh: rm: cannot remove '/etc/group': Permission denied
[cont-init   ] 10-init-users.sh: rm: cannot remove '/etc/shadow': Permission denied
[cont-init   ] 10-init-users.sh: terminated with error 1.

How can I run filebot wihtout running as root? Thanks

**Comment by jlesage** [comment]
Did you set the user via `USER_ID` and `GROUP_ID` environment variables ?

**Reply by pinkairfreshener** [reply]
services:
  filebot:
    image: jlesage/filebot:latest
    container_name: filebot
    user: "1002:1002"
    ports:
      - "5800:5800"
    volumes:
      - "/home/username/apps/filebot:/config:rw"
      - "/home/username:/data:rw"
    network_mode: host
    restart: unless-stopped

**Comment by jlesage** [comment, marked as answer]
Instead of `user: "1002:1002"`, try to set the user via `USER_ID` and `GROUP_ID` environment variables.

**Reply by pinkairfreshener** [reply]
perfect, that fixed it, thanks

**Reply by jlesage** [reply]
You are welcome!
